### ISSUE TITLE
shadow text vs persistent text

### ISSUE THREAD

**Issue by cranhandler** [contradb/contra#408]
> Data entering - all fields except the title are actually blank, but have the shadow text that disappears after you start typing, whereas Title remains until you delete it. Was that intentional?

**Comment by dcmorse** [comment]
Yeah, I guess it was. I mean, a good title of a dance is "untitled".

**Comment by dcmorse** [comment]
Accidentally closed.

**Comment by dcmorse** [comment]
Time to intentionally close this.
